easybuild/easyconfigs/a/abtem/abtem-1.0.0b24-foss-2020b.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PythonPackage'
+
+name = 'abtem'
+version = '1.0.0b24'
+
+homepage = 'https://imageio.github.io'
```

Comment:
wrong homepage URL, should be https://github.com/jacobjma/abTEM

Reply:
Fixed.